package-lock.json

```diff
@@ -64,7 +64,7 @@
     "@sindresorhus/slugify": {
       "version": "0.8.0",
       "resolved": "https://registry.npmjs.org/@sindresorhus/slugify/-/slugify-0.8.0.tgz",
-      "integrity": "sha512-Y+C3aG0JHmi4nCfixHgq0iAtqWCjMCliWghf6fXbemRKSGzpcrHdYxGZGDt8MeFg+gH7ounfMbz6WogqKCWvDg==",
+      "integrity": "sha1-VVC3+gZPOoqCZRRjrWNTeAVMctA=",
```

Comment:
Why would you even commit and push this?

Reply:
I did npm install using npm 5 as you suggested [here](https://github.com/MarkBind/markbind/pull/745#issuecomment-468982463), and this was the output generated.

Reply:
v5.10.0?

Reply:
Yes. On further investigation the issue seems to be reported [here](https://github.com/npm/npm/issues/17749), though for a different npm version.